packages/@aws-cdk/aws-lambda/lib/alias.ts

```diff
@@ -193,6 +208,26 @@ export class Alias extends QualifiedFunctionBase implements IAlias {
     });
   }
 
+  /**
+   * Configure provisioned concurrency autoscaling on a function alias. Returns a scalable attribute that can call
+   * `scaleOnUtilization()` and `scaleOnSchedule()`.
+   *
+   * @param props The properties for autoscaling
+   */
+  public autoScaleProvisionedConcurrency(props: EnableScalingProps): IScalableFunctionAttribute {
```

Comment:
We reserve the term "props" for constructors.

```suggestion
  public autoScaleProvisionedConcurrency(options: AutoScalingOptions): IScalableFunctionAttribute {
```